File: hello/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int row, coulmn, space, height;
    do
    {
        height = get_int("enter a number:");
    }
    while (height < 1 || height > 8);

    // NO.OF ROWS
    for (row = 0; row < height; row++)
    {
        // Spaces  -- form relation ship between no. of spaces , row and coulmns
        //  space = height - row - 1
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        // NO. OF # per coulmn
        for (coulmn = 0; coulmn <= row; coulmn++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int
while True:
    height = get_int("Enter the height:")
    if height < 1 or height > 8:
        print("Enter a height between 1 and 8:")
    else:
        break

for row in range(height):
    for space in range(height - row - 1):
        print(" ", end='')
    for hash in range(row + 1):
        print("#", end='')
    print()
